src/gui/properties/speedplotview.h

```diff
@@ -86,6 +90,19 @@ class SpeedPlotView : public QGraphicsView
     void paintEvent(QPaintEvent *event) override;
 
 private:
+    class Averager
+    {
+    public:
+        Averager(int divider, boost::circular_buffer<PointData> &sink);
+        inline void push(const PointData &p);
+        int cnt;
+
+    private:
+        int m_divider;
```

Comment:
It should be const (or template parameter).

Reply:
const applied